custom/icds_reports/utils/__init__.py

```diff
@@ -892,7 +892,11 @@ def create_aww_performance_excel_file(excel_data, data_type, month, state, distr
     for column_index in range(len(columns)):
         widths[columns[column_index]] = max(
             widths[columns[column_index]],
-            max(len(str(row[column_index])) for row in excel_data[1:]) * 4 // 3 if len(excel_data) >= 2 else 0
+            max(
+                len(row[column_index].decode('utf-8') if isinstance(row[column_index], bytes)
+                    else six.text_type(row[column_index])
+                    )
```

Comment:
Not sure if it matters, but this code had been counting the number of bytes and now it's counting the number of Unicode characters, which could be different. If you want to keep it the same as it was you'd change it to something like:
```py
len(
    row[column_index] if isinstance(row[column_index], bytes)
    else six.text_type(row[column_index]).encode('utf-8')
)
```

Reply:
Ya which was logically wrong because `x=b'ç'` would give 2 on len(x) but desired number should be 1 and `x=u'\u0c38'` would give 3 on len(six.text_type(x).encode('utf-8')) but in this case also x contains only 1 character. this length is used to set the width of the cell therefore should only count the characters to be displayed